package.json

```diff
@@ -52,7 +52,7 @@
     "jest-cli": "18.1.0",
     "minifyify": "7.3.4",
     "mkdirp": "0.5.1",
-    "node-sass": "3.13.0",
+    "node-sass": "^4.5.3",
```

Comment:
In practice, it wouldn't matter as installs would proceed from the lockfile.

However, updated for consistency.